README.md

```diff
@@ -1,5 +1,7 @@
 # CERF Anticipatory Action framework in Bangladesh
```

Comment:
In general really nice ReadMe! Maybe we could make a kind of ReadMe template for all our repos and use this as a base?